### PR TITLE
Add Kava Mainnet to ibc_info

### PR DESCRIPTION
**Pull request by samcode206**

### CODE REVIEW

File: ibc_info.json [hunk 1]
```diff
@@ -34,6 +34,13 @@
       "src_channel": "channel-4",
       "port_id": "wasm.sei1gjrrme22cyha4ht2xapn3f08zzw6z3d4uxx6fyy9zd5dyr3yxgzqqncdqn",
       "client_id": "07-tendermint-12"
+    },
+    {
+      "counterparty_chain_name": "kava_2222-10",
+      "dst_channel": "channel-132",
+      "src_channel": "channel-8",
```
Comment (LCyson) [inline ibc_info.json:41]: src_channel doesn't match with what's written here (channel-18) https://github.com/sei-protocol/chain-registry/blob/main/assetlist.json, can you double check?

**Review by LCyson — CHANGES_REQUESTED** [review]
@samcode206 ser can you also add asset metadata to here https://github.com/sei-protocol/chain-registry/blob/main/assetlist.json#L2

File: assetlist.json [hunk 2]
```diff
@@ -388,7 +388,29 @@
...
       },
       "coingecko_id": "tether"
-    }
+    },
+    {
+      "name": "USDT",
+      "description": "Tether USD₮ (Kava)",
+      "symbol":  "USD₮",
+      "base": "ibc/6C00E4AA0CC7618370F81F7378638AE6C48EFF8C9203CE1C2357012B440EBDB7",
+      "display": "USD₮",
```
Comment (besated) [inline assetlist.json:397]: The display denom here should match one of the denoms in `denom_units`. I suggest updating `erc20/tether/usdt` to `USD₮` in `denom_units` below

**Review by besated — CHANGES_REQUESTED** [review]
Thanks for making this pr! I left a comment, please take a look and let me know if you have any questions

File: assetlist.json (hunk 2 above)
Comment (samcode206) [reply assetlist.json:397]: I don't think `USD₮` is a denom unit it's just a display name, would that have any consequences or is it safe to change it still ?

Comment (besated) [reply assetlist.json:397]: It should be safe to change. The display field should match with one of the denoms in `denom_units`, and it's mainly used for determining the appropriate number of decimals/exponents to use for user interfaces and readability. It does not necessarily have to correspond to a transferrable denom. This aligns with the standards from the Cosmos chain registry

**Pull request closed without merging**